tools/src/core/reporter.py

```
@@ -0,0 +1,222 @@
+        if pr.updated:
+            lines.append('**Updated** (token limits):')
+            lines.append('')
+            for key, field_name, old_val, new_val in pr.updated:
+                lines.append(f'- `{key}` — `{field_name}`: {old_val} → {new_val}')
+            lines.append('')
+
+        if pr.deprecated:
+            lines.append('**Deprecated** (no longer in API):')
+            lines.append('')
+            for key in pr.deprecated:
+                lines.append(f'- `{key}`')
```

Ruff (0.15.9)

[warning] Use `list.extend` to create a transformed list

(PERF401)